### PR TITLE
OTA Update UI

### LINKED ISSUE #118: OTA Update UI
Check for update every minute or so, and add an alert just underneath the header if a new update is available, with the link to release notes and the button to install. 

After the user clicks "Install now" it should then install the update and keep the user updated with the progress. The node shouldn't be usable during this time.

### PR DESCRIPTION
**Pull request by mayankchhabra**
Closes #118

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -149,11 +162,55 @@ export default {
...
+            if (this.updateStatus.state === "success") {
+              this.$bvToast.toast(this.updateStatus.description, {
+                title: "Update successful",
+                autoHideDelay: 2000,
+                variant: "success",
+                solid: true,
+                toaster: "b-toaster-bottom-right"
+              });
+            } else if (this.updateStatus.state === "failed") {
+              this.$bvToast.toast(this.updateStatus.description, {
```
Comment (lukechilds) [inline src/App.vue:191]: Maybe dedupe this into a single call to toast() but just set the options in the if branches.

File: src/App.vue [hunk 2]
```diff
@@ -149,11 +162,55 @@ export default {
...
+                title: "Update failed",
+                autoHideDelay: 2000,
+                variant: "danger",
+                solid: true,
+                toaster: "b-toaster-bottom-right"
+              });
+            }
+
+            //refresh window to fetch latest code of dashboard
+            window.setTimeout(() => {
```
Comment (lukechilds) [inline src/App.vue:201]: You could use the delay util here.

File: src/App.vue (hunk 1 above)
Comment (mayankchhabra) [reply src/App.vue]: Yeahhhh, cleaner.

File: src/App.vue (hunk 2 above)
Comment (mayankchhabra) [reply src/App.vue]: Ooooh, of course ;)

**Pull request merged**